### PR TITLE
Created the mailer

### PR DESCRIPTION
**Pull request by solene2a**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_CLASS" value="AppKernel" />
+        <server name="APP_ENV" value="test" />
```
Comment (HeahDude) [inline phpunit.xml.dist]: This change does not belong to this PR :)

File: src/AppBundle/Contact/Mailer.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Please remove this useless comment.

File: src/AppBundle/Contact/Mailer.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ * User: SGUILLOT
+ * Date: 13/04/2018
+ * Time: 09:24
+ */
+
+namespace AppBundle\Contact;
+
+
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Extra empty line.

File: src/AppBundle/Contact/Mailer.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+class Mailer
+{
+
+	/*
+	 *
+	 */
+	protected $mailer;
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Why protected?

File: src/AppBundle/Contact/Mailer.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace AppBundle\Contact;
+
+
+class Mailer
+{
+
+	/*
+	 *
+	 */
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Please remove this lines.

File: src/AppBundle/Contact/Mailer.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+	protected $mailer;
+
+	function __construct(\Swift_Mailer $mailer)
+	{
+		$this->mailer = $mailer;
+	}
+
+
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Extra empty line.

File: src/AppBundle/Contact/Mailer.php [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+{
+
+	/*
+	 *
+	 */
+	protected $mailer;
+
+	function __construct(\Swift_Mailer $mailer)
+	{
+		$this->mailer = $mailer;
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Is this ident correct?

Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Is this indent correct?

File: src/AppBundle/Contact/Mailer.php [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+
+	/**
+	 * @param Message $message
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Remove this phpdoc.

File: src/AppBundle/Contact/Mailer.php [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+	 */
+	public function sendMail(Message $message){
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: `{` on a new line (PSR-2).


Missing return type hint.

File: src/AppBundle/Contact/Mailer.php [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+			$message->getContent()
+		);
+		$mail->setFrom($message->getEmail());
+		$mail->setTo('solene.guillot@sitec.fr');
+
+		return $this->mailer->send($mail);
+
+	}
+
+}
```
Comment (HeahDude) [inline src/AppBundle/Contact/Mailer.php]: Missing empty line at end of file (EOF).

File: src/AppBundle/Controller/MainController.php [hunk 11]
```diff
@@ -46,15 +47,5 @@ public function contactAction(Request $request): Response
...
-        $mail = \Swift_Message::newInstance(
-            $message->getSubject(),
-            $message->getContent()
-        );
-        $mail->setFrom($message->getEmail());
-        $mail->setTo('solene.guillot@sitec.fr');
-
-        return $this->get('mailer')->send($mail);
-    }
+
```
Comment (HeahDude) [inline src/AppBundle/Controller/MainController.php]: Extra empty line :).

File: tests/AppBundle/Controller/MainControllerTest.php [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+ * User: SGUILLOT
+ * Date: 12/04/2018
+ * Time: 17:45
+ */
+
+namespace Tests\AppBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class MainControllerTest extends WebTestCase
```
Comment (HeahDude) [inline tests/AppBundle/Controller/MainControllerTest.php]: This file does not belong to this PR.

File: tests/AppBundle/Game/GameContextTest.php [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+namespace Tests\AppBundle\Game;
+
+
+use AppBundle\Game\GameContext;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+class GameContextTest extends TestCase
```
Comment (HeahDude) [inline tests/AppBundle/Game/GameContextTest.php]: This file does not belong to this PR.

File: tests/AppBundle/Game/GameTest.php [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace Tests\AppBundle\Game;
+
+use AppBundle\Game\Game;
+use AppBundle\Game\GameContext;
+use PHPUnit\Framework\TestCase;
+
+
+
+class GameTest extends TestCase
```
Comment (HeahDude) [inline tests/AppBundle/Game/GameTest.php]: This file does not belong to this PR.

File: phpunit.xml.dist (hunk 1 above)
Comment (solene2a) [reply phpunit.xml.dist]: fixed

File: src/AppBundle/Contact/Mailer.php (hunk 2 above)
Comment (solene2a) [reply src/AppBundle/Contact/Mailer.php]: fixed

File: src/AppBundle/Contact/Mailer.php (hunk 3 above)
Comment (solene2a) [reply src/AppBundle/Contact/Mailer.php]: fixed

File: src/AppBundle/Contact/Mailer.php (hunk 4 above)
Comment (solene2a) [reply src/AppBundle/Contact/Mailer.php]: fixed in private

File: src/AppBundle/Contact/Mailer.php (hunk 8 above)
Comment (HeahDude) [reply src/AppBundle/Contact/Mailer.php]: You didn't remove this phpdoc :)

File: src/AppBundle/Contact/Mailer.php (hunk 10 above)
Comment (solene2a) [reply src/AppBundle/Contact/Mailer.php]: fixed

File: src/AppBundle/Controller/MainController.php (hunk 11 above)
Comment (solene2a) [reply src/AppBundle/Controller/MainController.php]: fixed

File: src/AppBundle/Contact/Mailer.php (hunk 5 above)
Comment (solene2a) [reply src/AppBundle/Contact/Mailer.php]: fixed

File: tests/AppBundle/Controller/MainControllerTest.php (hunk 12 above)
Comment (solene2a) [reply tests/AppBundle/Controller/MainControllerTest.php]: fixed removed

File: tests/AppBundle/Game/GameContextTest.php (hunk 13 above)
Comment (solene2a) [reply tests/AppBundle/Game/GameContextTest.php]: fixed removed

File: tests/AppBundle/Game/GameTest.php (hunk 14 above)
Comment (solene2a) [reply tests/AppBundle/Game/GameTest.php]: fixed removed
